docs/compute/drivers/upcloud.rst

```diff
@@ -0,0 +1,31 @@
+UpCloud Driver Documentation
+===============================
+`UpCloud`_ is a Finnish IaaS provider offering the most high performance 
+servers with extremely redundant infrastructure to the most demanding customers.
```

Comment:
Can we be more objective? You can look at the pages from other providers. Thanks!

Reply:
Revised the wording there and updated the PR accordingly :)